### ISSUE TITLE
Build fails due to swift-tools-support-core

### ISSUE THREAD

**Issue by chkpnt** [apple/swift-format#430]
I'm trying to build 0.50700.1 on macOS with Xcode 14.0.1 / Swift 5.7.0 as described in the README.md.

```
> swift build -c release
warning: Usage of /Users/ich/Library/org.swift.swiftpm/collections.json has been deprecated. Please delete it and use the new /Users/ich/Library/org.swift.swiftpm/configuration/collections.json instead.
Updating https://github.com/apple/swift-tools-support-core.git
Updating https://github.com/apple/swift-syntax
Updating https://github.com/apple/swift-argument-parser.git
Updated https://github.com/apple/swift-argument-parser.git (6.19s)
Updated https://github.com/apple/swift-syntax (6.19s)
Updated https://github.com/apple/swift-tools-support-core.git (6.19s)
Computing version for https://github.com/apple/swift-tools-support-core.git
error: Dependencies could not be resolved because no versions of 'swift-tools-support-core' match the requirement 0.2.7..<0.3.0 and root depends on 'swift-tools-support-core' 0.2.7..<0.3.0.
```

Is there anything I'm missing?

**Comment by chkpnt** [comment]
Mh, maybe it is a network issue: At home without a corporate proxy it's building fine. But I'm confused, as I ruled out the proxy as the culprit, because I was able to build `swift-format` via `brew install --build-from-source`.

Will try it again tomorrow at work.

**Comment by jesus-mg-ios** [comment]
@chkpnt this issue could be resolved with  #427

**Comment by chkpnt** [comment]
Figured it out: Usually, git respects the environment variables `http_proxy` and `https_proxy`. But `swift build` doesn't seem to provide these environment variables to the git-subprocess. Hence you **must** configure your http proxy in `~/.gitconfig` (resp. `git config --global http.proxy ...`).
